Documentation/conf.py

```diff
@@ -117,12 +117,13 @@
 .. |SCM_ARCHIVE_NAME| replace:: \{a}
 .. |SCM_ARCHIVE_FILENAME| replace:: \{f}
 .. |SCM_ARCHIVE_LINK| replace:: \{l}
+.. |RELEASE| replace:: \{r}
```

Comment:
It looks like [Sphinx has you covered](https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#substitutions) already. I think `|release|` should be available and does the same, in which case there's no need to add an upper-case clone.

Reply:
I see, I tried that first time, but didn't know I had to use extra surrounding chars for the expansion to work.